File: homeworks/Naira_Poghosyan/InvertNumber/InvertNumber.cpp
//Author: Naira
#include <iostream>

int invertNumber(int i_tiv){	

    static int i_invertNumber = 0; 
    if (i_tiv==0){return i_invertNumber;}
    i_invertNumber = i_invertNumber * 10 + i_tiv % 10;	
    invertNumber(i_tiv/10);

}

int main(){

    int i_inputNumber;
    std::cout << "Please input the number ..." <<'\n';
    std::cin >> i_inputNumber;
    int i_outputNamber=invertNumber(i_inputNumber);
    std::cout << "Results :  " <<i_outputNamber<<'\n';
    return 0;
}

File: homeworks/Naira_Poghosyan/Tasks14.06/src/example/am/InvertNumber.java
package example.am;

public class InvertNumber {
	
	static int i_invertNumber = 0;
	int getInvertNumber(int i_tiv){
		
		if (i_tiv==0){return i_invertNumber;}
		i_invertNumber = i_invertNumber * 10 + i_tiv % 10;
		return getInvertNumber(i_tiv/10);
		
	}

}
